Jojoucsd/tic-tac-toe/ticTacToe.js

```diff
@@ -10,6 +10,8 @@ $(document).ready(function () {
   
   var player = "X"; //first player
   var counts = 0; //how many moves on the field
+  var addIdx = 0;
+  var addIdo = 0;
```

Comment:
Travis: I am still working on the ticTacToe trying to add new feature and new logic to the app. I believe the code right now is broken at the moment. Just when I pushed the HW up to the git it got pushed as well. I will fix it during the weekend. Thanks 
Ling
> On Oct 16, 2015, at 11:11 AM, tgaff <notifications@github.com <mailto:notifications@github.com>> wrote:
> 
> In Jojoucsd/tic-tac-toe/ticTacToe.js <https://github.com/sf-wdi-22-23/23-homework/pull/97#discussion_r42272613>:
> 
> > @@ -10,6 +10,8 @@ $(document).ready(function () {
> >    
> >    var player = "X"; //first player
> >    var counts = 0; //how many moves on the field
> > +  var addIdx = 0;
> > +  var addIdo = 0;
> changes to tictactoe? Is this intentional? @Jojoucsd <https://github.com/Jojoucsd>
> —
> Reply to this email directly or view it on GitHub <https://github.com/sf-wdi-22-23/23-homework/pull/97/files#r42272613>.
>